MULTI-TARGET-BUILD.md

```diff
@@ -0,0 +1,237 @@
+# Multi-Target Build Support
```

Comment:
I don't think we need this file or the .bat or .ps1 scripts since multi-target building is a commonly used .NET functionality.